GenderPayGap.Database/Migrations/20200505085232_CreateViewsForReportsReader.cs

```diff
@@ -1,12 +1,737 @@
+                + " FROM dbo.UserStatus AS ust "
+                + " INNER JOIN dbo.Users AS usr ON usr.UserId = ust.UserId "
+                + " INNER JOIN dbo.Users AS modifyingUser ON modifyingUser.UserId = ust.ByUserId ");
+
+            migrationBuilder.Sql(" GRANT SELECT " + " 	ON OBJECT::dbo.[UserStatusInfoView] " + " 	TO [ReportsReaderDv]; ");
+
+            #endregion
+
+            #region 201812131233344_Add function OrganisationStatusIdToString.cs
+
+            migrationBuilder.Sql(
+                " CREATE FUNCTION [dbo].[OrganisationStatusIdToString]"
+                + " ("
+                + " 	@organisationStatusId tinyint"
+                + " )"
+                + " RETURNS varchar(20)"
+                + " AS"
+                + " BEGIN"
+                + " 	DECLARE @Result varchar(20)"
+                + " "
+                + " 	SELECT @Result = CASE @organisationStatusId "
+                + " 						WHEN 0 THEN 'Unknown' "
+                + " 						WHEN 1 THEN 'New' "
+                + " 						WHEN 2 THEN 'Suspended' "
+                + " 						WHEN 3 THEN 'Active' "
+                + " 						WHEN 4 THEN 'Retired' "
+                + " 						WHEN 5 THEN 'Pending' "
+                + " 						WHEN 6 THEN 'Deleted' "
+                + " 						ELSE 'Error' "
+                + " 					END + ' (' + CAST(@organisationStatusId AS VARCHAR(3)) + ')'"
+                + " "
+                + " 	RETURN @Result		"
+                + " END ");
+
+            migrationBuilder.Sql(" GRANT EXECUTE " + " 	ON OBJECT::[dbo].[OrganisationStatusIdToString] " + " 	TO [ReportsReaderDv]; ");
+
+            #endregion
+
+            #region 201812131245269_Add function OrganisationSectorTypeIdToString.cs
+
+            migrationBuilder.Sql(
+                " CREATE FUNCTION [dbo].[OrganisationSectorTypeIdToString]"
+                + " ("
+                + " 	@organisationSectorTypeId tinyint"
+                + " )"
+                + " RETURNS varchar(20)"
+                + " AS"
+                + " BEGIN"
+                + " 	DECLARE @Result varchar(20)"
+                + " "
+                + " 	SELECT @Result = CASE @organisationSectorTypeId"
+                + " 						WHEN 0 THEN 'Unknown' "
+                + " 						WHEN 1 THEN 'Private' "
+                + " 						WHEN 2 THEN 'Public' "
+                + " 						ELSE 'Error' "
+                + " 					END + ' (' + CAST(@organisationSectorTypeId AS VARCHAR(3)) + ')'"
+                + " "
+                + " 	RETURN @Result"
+                + " "
+                + " END ");
+
+            migrationBuilder.Sql(" GRANT EXECUTE " + " 	ON OBJECT::[dbo].[OrganisationSectorTypeIdToString] " + " 	TO [ReportsReaderDv]; ");
+
+            #endregion
+
+            #region 201812131250589_Modify view OrganisationInfoView.cs
+
+            migrationBuilder.Sql(
+                " ALTER VIEW [dbo].[OrganisationInfoView] "
```

Comment:
This `ALTER VIEW` replaces the view created on line 66.
I think we can get rid of the block at line 66 and change this block to use the `CREATE VIEW` syntax (which might be as simple as switching `ALTER` for `CREATE`)